Source/Demos/Demo.PrimitiveBatch/Game1.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using MonoGame.Extended.Graphics;
+using MonoGame.Extended.Graphics.Batching;
+using SpriteBatch = MonoGame.Extended.Graphics.Batching.SpriteBatch;
```

Comment:
`PrimitiveSpriteBatch` maybe. Or `SpriteBatchExtended`. I dunno, what would you say is the primary thing it does different to the normal sprite batch?